blocks/library/text/index.js

```diff
@@ -70,12 +67,10 @@ registerBlockType( 'core/text', {
 		const { align, content } = attributes;
 
 		if ( ! align ) {
-			return content;
+			return <p>{ content }</p>;
 		}
 
-		return Children.map( content, ( paragraph ) => (
-			cloneElement( paragraph, { style: { textAlign: align } } )
-		) );
+		return <p style={ { textAlign: align } }>{ content }</p>;
```

Comment:
Well, eventually, yes. For now since the pragma is defined as `wp.element.createElement`, it wouldn't be used anyways. Let's just leave it as-is for now, we can handle it separately.